src/nanorc/core.py

```diff
@@ -77,8 +79,11 @@ def __init__(self, console: Console, top_cfg: str, partition_label:str, run_num_
         self.timeout = timeout
         self.return_code = None
         self.logbook = None
+        self.logbook_type = logbook_type
         self.log_path = None
-
+        self.current_run_type = None
```

Comment:
You shouldn't need to save the current run type, this is in `self.runs[-1].run_type`

Reply:
Actually, I think you don't need this at all, since this is only used on `start`.